osu.Game.Rulesets.Osu/Skinning/Default/SpinnerRotationTracker.cs

```diff
@@ -107,9 +112,74 @@ public void AddRotation(float angle)
             }
 
             currentRotation += angle;
+
```

Comment:
Unrelated to this PR's changes, but `rotationTransferred` usage above needs explanation because I have no idea what it's doing.